### PR TITLE
Merge xrpl_style into master

### PR DESCRIPTION
**Pull request by ryangyoung**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,10 +1,12 @@
-The Ripple Dev Portal and all its original contents are provided under the MIT License. Additional libraries and resources used by the Ripple Dev Portal are subject to their own licenses, as described below.
+The XRP Ledger Dev Portal and all its original contents are provided under the MIT License. Additional libraries and resources used by the XRP Ledger Dev Portal are subject to their own licenses, as described below.
 
 The MIT License (MIT)
 
-Ripple Dev Portal Copyright (c) 2014, 2015 Ripple Labs, Inc.
-Flatdoc Copyright (c) 2013, 2014, Rico Sta. Cruz.
+XRP Ledger Dev Portal Copyright (c) 2019 XRP Ledger Project
```
Comment (MarkusTeufelberger) [inline LICENSE:5]: Interesting, who is the "XRP Ledger Project" and who had the copyright on this documentation since 2014?

**Comment by mDuo13** [comment]
Rebased & fixed merge conflicts

**Comment by mDuo13** [comment]
I am not sure why this is happening:

```
23:58:52 Checking Target: data-api-only
23:58:53   File "/usr/local/lib/python3.5/dist-packages/dactyl/dactyl_build.py", line 580, in render_page
23:58:53     bypass_errors=bypass_errors,
23:58:53   File "/usr/local/lib/python3.5/dist-packages/dactyl/dactyl_build.py", line 227, in parse_markdown
23:58:53     logger=logger,
23:58:53   File "tool/filter_external_links.py", line 31, in filter_soup
23:58:53     link['class'] = link.get('class',[]) + ['external-link']
23:58:53 Error when fetching page Ripple Data API v2: TypeError("Can't convert 'list' object to str implicitly",)
23:58:53 Error building this target; skipping link checker.
```

Per the BeautifulSoup documentation, `link['class']` should always be a list, so this TypeError shouldn't occur. And on my local system, that is indeed how it works.

**Comment by mDuo13** [comment]
This PR fixes #390 .

**Review by mDuo13 — APPROVED** [review]
sbhj_making_it_happen

**Pull request merged**